### PR TITLE
Build Aarch64 executable on centos:8

### PR DESCRIPTION
**Pull request by Jongy**
Attempts to fix what was introduced in https://github.com/Granulate/gprofiler/pull/296 (using grpcio built/downloaded on `centos:7` fails miserably)

```
Traceback (most recent call last):
  File "pyi_build.py", line 11, in <module>
    from gprofiler.main import main
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "gprofiler/main.py", line 25, in <module>
    from gprofiler import __version__, merge
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "gprofiler/merge.py", line 13, in <module>
    from gprofiler.containers_client import ContainerNamesClient
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "gprofiler/containers_client.py", line 7, in <module>
    from granulate_utils.containers.client import ContainersClient
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "granulate_utils/containers/client.py", line 9, in <module>
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "granulate_utils/containers/cri.py", line 9, in <module>
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "grpc/__init__.py", line 22, in <module>
  File "/usr/local/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "grpc/_compression.py", line 15, in <module>
ImportError: /tmp/_MEILFXli1/grpc/_cython/cygrpc.cpython-36m-aarch64-linux-gnu.so: undefined symbol: _ZTVNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE
[115121] Failed to execute script pyi_build
```

This PR switches the ARM build to `centos:8` which should be okay I presume.

### CODE REVIEW

File: pyi.Dockerfile [hunk 1]
```diff
@@ -155,6 +162,10 @@ COPY granulate-utils/granulate_utils granulate-utils/granulate_utils
 RUN python3 -m pip install -r requirements.txt
 
 COPY exe-requirements.txt exe-requirements.txt
+RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
```
Comment (Jongy) [inline pyi.Dockerfile:165]: Document

File: pyi.Dockerfile [hunk 2]
```diff
@@ -100,6 +100,14 @@ RUN ./bcc_helpers_build.sh
 # bcc & gprofiler
 FROM centos${GPROFILER_BUILDER} AS build-stage
 
+# fix repo links for CentOS 8, and enable powertools (required to download glibc-static)
+RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
```
Comment (adi-benz) [inline pyi.Dockerfile:104]: Docs in lines 17-19 in this file is somewhat missleading.

Besides, if we have differences in the container versions we pass to `pyi.Dockerfile` between x86_64 and aarch64, then maybe not have a default for them in `pyi.Dockerfile` and have the comment and sha256 of the container in `build_x86_64_container.sh`?

**Review by adi-benz — CHANGES_REQUESTED** [review]

File: pyi.Dockerfile (hunk 2 above)
Comment (Jongy) [reply pyi.Dockerfile:104]: Fair

Comment (Jongy) [reply pyi.Dockerfile:94]: Fixed & tested build before / after, seems okay (result is cached so I didn't fuck up any base layer etc)

**Review by adi-benz — APPROVED** [review]

**Comment by Jongy** [comment]
Tested again on aarch64, all is well

**Pull request merged**